xc/xc_resources.ml

```diff
@@ -30,6 +30,8 @@ let gimtool = ref "/opt/xensource/bin/gimtool"
 
 let alternatives = ref "/usr/lib/xapi/alternatives"
 
+let usb_reset_script = ref "/opt/xensource/libexec/usb_reset.py"
```

Comment:
Does this have to be a reference? It is never updated.

Reply:
hmm, yes, this will not be updated.

Reply:
Ah, the others are references because we have configuration parameters for them - ie you can set the script in xapi.conf